lib/authorize.js

```diff
@@ -0,0 +1,55 @@
+import { getLogger } from 'appium-logger';
+import { exec } from 'teen_process';
+import { getPath } from 'appium-xcode';
+import path from 'path';
+
+export async function authorize(insecure) {
+
+  let logger = getLogger('authorize');
+  let xcodeDir;
+  let user;
+
+  try { 
+    // enable developer tools
+    logger.info('Enabling DevToolsSecurity');
+    await exec('DevToolsSecurity --enable');
+
+    // update security db -- removes authorization prompt
+    logger.info(`Updating security db for ${insecure ? 'insecure' :
```

Comment:
I'd break this into two strings outside of the interpolated js, and line up under the column the first line's string starts on. (sounds confusing, I know ;-))